scripts/spark/putFilesFromLocalToHDFS.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-SPARK_RESOURCES_DIR="/spark_volume/resources"
+SPARK_RESOURCES_DIR="/spark_volume/resourcesGenerators"
```

Comment:
use camel case _/sparkVolume/resourcesGenerators_, or snake case  _/spark_volume/resources_generators_ in the **SPARK_RESOURCES_DIR**.